pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/PresentationServiceImpl.scala

```diff
@@ -610,6 +596,20 @@ private class PresentationServiceImpl(
     } yield (schemaId, anoncredLibSchema)
   }
 
+  private def resolveCredentialDefinition(
+      credentialId: String
+  ): IO[UnexpectedError, (String, AnoncredCredentialDefinition)] = {
+    for {
+      uri <- ZIO.attempt(new URI(credentialId)).mapError(e => UnexpectedError(e.getMessage))
+      content <- uriDereferencer.dereference(uri).mapError(e => UnexpectedError(e.error))
+      _ <-
+        PublicCredentialDefinitionSerDesV1.schemaSerDes
+          .validate(content)
+          .mapError(error => UnexpectedError(s"AnonCreds Schema parsing error: $error"))
+      anoncredCredentialDefinition = AnoncredCredentialDefinition(content)
+    } yield (credentialId, anoncredCredentialDefinition)
```

Comment:
Why do we need to yield the `credentialId` if it's an input parameter for the method?

Reply:
need to rename it.

Reply:
fixed